config/filter.d/postfix.conf

```diff
@@ -52,6 +52,10 @@ mdre-aggressive = %(mdre-auth2)s
 mdpr-errors = too many errors after \S+
 mdre-errors = ^from [^[]*\[<HOST>\]%(_port)s$
 
+# Extra mode "screendnsbl", triggered on postfix/postscreen[<PID>]: DNSBL rank <NUM> for [<HOST>]:<PORT>
+mdpr-screendnsbl = DNSBL rank \d+
+mdre-screendnsbl = for \[<HOST>\]%(_port)s
```

Comment:
The resulting failregex would be unanchored, what is no go for the PRs.
Please change:
```diff
mdre-screendnsbl = for \[<HOST>\]%(_port)s
mdre-screendnsbl = ^for \[<HOST>\]%(_port)s
```

Reply:
fixed